Mac/BuildScript/build-installer.py

```diff
@@ -1280,29 +1259,24 @@ def buildPython():
     if shared_lib_error:
         fatal("Unexpected shared library errors.")
 
-    if PYTHON_3:
-        LDVERSION=None
-        VERSION=None
-        ABIFLAGS=None
-
-        fp = open(os.path.join(buildDir, 'Makefile'), 'r')
-        for ln in fp:
-            if ln.startswith('VERSION='):
-                VERSION=ln.split()[1]
-            if ln.startswith('ABIFLAGS='):
-                ABIFLAGS=ln.split()
-                ABIFLAGS=ABIFLAGS[1] if len(ABIFLAGS) > 1 else ''
-            if ln.startswith('LDVERSION='):
-                LDVERSION=ln.split()[1]
-        fp.close()
+    LDVERSION = None
+    VERSION = None
+    ABIFLAGS = None
+
+    fp = open(os.path.join(buildDir, 'Makefile'), 'r')
+    for ln in fp:
+        if ln.startswith('VERSION='):
+            VERSION=ln.split()[1]
+        if ln.startswith('ABIFLAGS='):
+            ABIFLAGS=ln.split()
+            ABIFLAGS=ABIFLAGS[1] if len(ABIFLAGS) > 1 else ''
+        if ln.startswith('LDVERSION='):
+            LDVERSION=ln.split()[1]
+    fp.close()
```

Comment:
Might as well modernise this 12-year-old code while we're here?

```suggestion
    with open(os.path.join(buildDir, 'Makefile'), 'r') as fp:
        for ln in fp:
            if ln.startswith('VERSION='):
                VERSION=ln.split()[1]
            if ln.startswith('ABIFLAGS='):
                ABIFLAGS=ln.split()
                ABIFLAGS=ABIFLAGS[1] if len(ABIFLAGS) > 1 else ''
            if ln.startswith('LDVERSION='):
                LDVERSION=ln.split()[1]
```